plugin-template/android/build.gradle

```diff
@@ -1,5 +1,6 @@
 ext {
     junitVersion =  project.hasProperty('junitVersion') ? rootProject.ext.junitVersion : '4.12'
+    androidxAppCompatVersion = project.hasProperty('androidxAppCompatVersion') ? rootProject.ext.androidxAppCompatVersion : '1.1.0'
```

Comment:
```suggestion
    androidxAppCompatVersion = project.hasProperty('androidxAppCompatVersion') ? rootProject.ext.androidxAppCompatVersion : '1.2.0'
```

Reply:
I think so. Without this dependency, Android Studio doesn't resolve basic `Activity.java` methods that can be overridden like `runOnUiThread` and it isn't clear how to fix that. Any plugin that tries to run code that impacts the UI will need to use this method. Dan came across this when working on a plugin being broken out of core capacitor.